projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-form/admin-badges-form.component.html

```diff
@@ -0,0 +1,36 @@
+<nb-card class="admin-badges-form" >
```

Comment:
Why was this html file formatted with 4 spaces instead of 2 spaces?

Reply:
done with 2 spaces